pbj-core/pbj-runtime/src/main/java/com/hedera/pbj/runtime/io/buffer/UnsafeUtils.java

```diff
@@ -0,0 +1,295 @@
+package com.hedera.pbj.runtime.io.buffer;
+
+import java.lang.reflect.Field;
+import java.nio.Buffer;
+import java.nio.ByteBuffer;
+import java.nio.ByteOrder;
+import java.security.AccessController;
+import java.security.PrivilegedExceptionAction;
+
+final class UnsafeUtils {
```

Comment:
Missing Tests for this class. The whole idea of a separate class to do unsafe operations is we can test it like crazy till we are 100% sure it is rock solid. The more testing the better, be brutal. A mistake here could kill a node in the network!

Reply:
Added basic tests. We should expand with more when usages get better defined.